test/e2e/specs/all-features-confirmation/navbar/no-logo.spec.js

```diff
@@ -140,11 +143,12 @@ describe('Navbar ', function() {
             });
 
             it('first level nested link should inherit newTab value from parent when newTab is undefined and navigate in new tab', function (done) {
-                var recordsetsMenu = menu.all(by.tagName('li')).get(1);
+                // Added chaise-nav-item class to get all first level menu item in navbar
+                var recordsetsMenu = menu.all(by.css('.chaise-nav-item')).get(1);
                 expect(recordsetsMenu.getText()).toBe("RecordSets", "Second top level menu option text is incorrect");
 
                 recordsetsMenu.click().then(function () {
-                    var datasetOption = recordsetsMenu.all(by.tagName('li')).get(0);
+                    var datasetOption = recordsetsMenu.all(by.css('.dropdown-item')).get(0);
```

Comment:
This works because each dropdown item is just a link, but this class isn't always present on every dropdown item. I think we want to do `by.tagName('a')` here.